Directory.Build.props

```diff
@@ -50,8 +50,8 @@
 
   <!-- The TFMs to build and test against. -->
   <PropertyGroup>
-    <!-- Naming is based on dotnet/runtime one -->
-    <NetFrameworkMinimum>net462</NetFrameworkMinimum>
+    <!-- When not building from source, hardcode the NetCurrent TFM value as it is hardcoded in a number of places, i.e. nuspec files. -->
+    <NetCurrent Condition="'$(DotNetBuildSourceOnly)' != 'true'">net9.0</NetCurrent>
```

Comment:
The change does not look right, did you mean to add the property rather than replacing .net framework?

```suggestion
    <!-- Naming is based on dotnet/runtime one -->
    <NetFrameworkMinimum>net462</NetFrameworkMinimum>
    <!-- When not building from source, hardcode the NetCurrent TFM value as it is hardcoded in a number of places, i.e. nuspec files. -->
    <NetCurrent Condition="'$(DotNetBuildSourceOnly)' != 'true'">net9.0</NetCurrent>
```

Reply:
The NetFrameworkMinimum property is already provided by the Arcade SDK: https://github.com/dotnet/arcade/blob/d5c8bb8726b46b5867961f5d8f56f1b13e72dcb9/src/Microsoft.DotNet.Arcade.Sdk/tools/TargetFrameworkDefaults.props#L27

No need to define it here